src/compiler/typescript_to_rust/azle_generate/src/main.rs

```diff
@@ -14,11 +16,31 @@ fn main() {
     let compiler_info_string = std::fs::read_to_string(compiler_info_path).unwrap();
     let compiler_info: CompilerInfo = serde_json::from_str(&compiler_info_string).unwrap();
 
+    let environment_variables = if let Some(envs) = args.get(2) {
```

Comment:
Let's leave a comment here, consider reading the dfx.json file, remove as many command line arguments as possible